util/util.go

```diff
@@ -144,3 +145,33 @@ func Contains(l []string, s string) bool {
 
 	return false
 }
+
+func FormatBytes(bytes int) (float64, string) {
```

Comment:
I think we'd better to use something existing, like: `"github.com/docker/go-units"`

Reply:
Switched!